tools/testbench/file.c

```diff
@@ -482,6 +475,7 @@ static struct comp_dev *file_new(const struct comp_driver *drv,
 		}
 		break;
 	case FILE_WRITE:
+		dev->ipc_config.type = SOF_COMP_FILEWRITE;
```

Comment:
And also this. Need to keep type as HOST and DAI. Remove of these lines fixes the issue.